docs/en/quantization/w4a16.md

```diff
@@ -1,92 +1,130 @@
-# W4A16 LLM Model Deployment
+# INT4 Weight-only Quantization and Deployment (W4A16)
 
-LMDeploy supports LLM model inference of 4-bit weight, with the minimum requirement for NVIDIA graphics cards being sm80, such as A10, A100, Geforce 30/40 series.
+LMDeploy adopts [AWQ](https://arxiv.org/abs/2306.00978) algorithm for 4bit weight-only quantization. By developed the high-performance cuda kernel, the 4bit quantized model inference achieves up to 2.4x faster than FP16.
 
-Before proceeding with the inference, please ensure that lmdeploy is installed.
+LMDeploy supports the following NVIDIA GPU for W4A16 inference:
+
+- Turing(sm75): 20 series, T4
+
+- Ampere(sm80,sm86): 30 series, A10, A16, A30, A100
+
+- Ada Lovelace(sm90): 40 series
+
+Before proceeding with the quantization and inference, please ensure that lmdeploy is installed.
 
 ```shell
 pip install lmdeploy[all]
 ```
 
-## 4-bit LLM model Inference
+This article comprises the following sections:
+
+<!-- toc -->
 
-You can download the pre-quantized 4-bit weight models from LMDeploy's [model zoo](https://huggingface.co/lmdeploy) and conduct inference using the following command.
+- [Quantization](#quantization)
+- [Evaluation](#evaluation)
+- [Inference](#inference)
+- [Performance](#performance)
+- [Service](#service)
 
-Alternatively, you can quantize 16-bit weights to 4-bit weights following the ["4-bit Weight Quantization"](#4-bit-weight-quantization) section, and then perform inference as per the below instructions.
+<!-- tocstop -->
 
-Take the 4-bit Llama-2-chat-7B model from the model zoo as an example:
+## Quantization
+
+A single command execution is all it takes to quantize the model. The resulting quantized weights are then stored in the $WORK_DIR directory.
 
 ```shell
-git-lfs install
-git clone https://huggingface.co/lmdeploy/llama2-chat-7b-w4
+lmdeploy lite auto_awq \
+  --model $HF_MODEL \                # Model name or path, either model repo name on huggingface hub like 'internlm/internlm-chat-7b', or a model path in local host
```

Comment:
remove `--model` since it is positional argument.

Reply:
fixed